platform/linux-generic/odp_queue_lf.c

```diff
@@ -28,57 +28,8 @@ static inline void atomic_zero_u128(u128_t *atomic)
 	__atomic_store_n(atomic, 0, __ATOMIC_RELAXED);
 }
 
-#if defined(__aarch64__)
-/* ARMv8 has atomic load-acq/store-rel instructions for a pair of
- * 64bit of data. GCC atomic built-in for 128bits does not utilize these
- * instructions but uses locks instead. Override GCC built-in for ARMv8.
- */
 #include <odp_cpu.h>
 
-static inline int atomic_cas_acq_rel_u128(u128_t *atomic, u128_t old_val,
```

Comment:
Instead of duplicating functionality, couldn't the new 128-bit atomic API functions be used here?

Reply:
I will look into it. Thanks!